examples/plugins/example-plugin/CMakeLists.txt

```diff
@@ -6,6 +6,12 @@ include(GNUInstallDirs)
 
 find_package(f3d REQUIRED)
 
+# Check Plugin SDK is available
+if(NOT (DEFINED f3d_SDK_PLUGIN AND ${f3d_SDK_PLUGIN}))
```

Comment:
Can we do something at the SDK level? I don't expect users to check that manually

Reply:
if the user do not check, it will just fail on the next call.

What we could do is to have a component logic on f3dConfig, but I'm not sure what our component will be and how to implement it.

In any case, user will still need to do something like:

`find_package(f3d REQUIRED COMPONENTS sdk plugin)`

Reply:
What about raising a fatal error in `f3d_init_plugin` if the plugin SDK is not present?

Reply:
f3d_init_plugin is F3DPlugin.cmake, which is not installed

Reply:
ok then